lib/auth/auth.go

```diff
@@ -459,6 +463,24 @@ func (a *Server) runPeriodicOperations() {
 	defer ticker.Stop()
 	defer heartbeatCheckTicker.Stop()
 	defer promTicker.Stop()
+
+	firstReleaseCheck := utils.FullJitter(time.Hour * 6)
+
+	// this environment variable is "unstable" since it will be deprecated
+	// by an upcoming tctl command. currently exists for testing purposes only.
+	if os.Getenv("TELEPORT_UNSTABLE_VC_SYNC_ON_START") == "yes" {
```

Comment:
Do you intend to commit this here or will it be removed before merging?

Reply:
Yeah, I intend to commit it.  I think its important to have some way to manually test the functionality in-situ without having to leave a cluster on overnight.  Used the `UNSTABLE` naming scheme to communicate that fact that this won't be a permanent part of the API.  Once the `tctl version-control` family of commands land, we'll have something like `tctl version-control sync` that should let folks manually trigger a pull instead.